spec/spec_helper.rb

```diff
@@ -107,7 +107,7 @@ def load_fixture(fixture)
   end
 
   config.after(:suite) do
-    Datadog.tracer.shutdown!
```

Comment:
no need to shutdown anymore ? or should it still be called after ?

Reply:
yes, it will be call at exit

Reply:
we need to make sure that the worker buffer is empty instead.